src/docs/product/dashboards/widget-builder/index.mdx

```diff
@@ -0,0 +1,125 @@
+
+</Note>
+
+When adding a widget or editing an existing one, you will be taken to a new page where you can see the dashboard widget builder, as shown below:
+
+![Widget Builder overview.](widget-builder-overview.png)
+
+The widget builder has several configuration options that you can use to shape data and add information to your dashboard. These options are presented as steps in the widget builder, but you don't have to do them in the order they're presented in.
+
+Some of these options are standard and displayed for all widgets, but others are only displayed if you make certain selections. Below is a list of all the options that can be displayed when configuring your widget.
+
+## Widget Title
+
+By default, all new widgets are titled “Custom Widget”, but you can edit the title by clicking on the text. We recommend you update the title.
+
+## Choose your visualization
+
+In this step, you can choose from four different types. They are:
+
+### Area Chart / Bar Chart / Line Chart
+
+Area, bar, and line chart visualizations allow you to plot one or more aggregate functions as a time series.
+
+![Time-series visualizations.](widget-builder-time-series-visualizations.png)
+
+Each chart type allows you to solve different problems: 
+
+- Area charts are ideal for showing results that are cumulative or where you want to show a breakdown. An example would be “count of errors for each major browser”. 
+- Bar charts will group results by day, making them good for daily roll-ups or as “big picture” summaries. An example would be “count of errors per day”. 
+- Line charts are a general-purpose visualization. Make sure the Y-Axis units match if you add multiple overlays. Mixing count with duration will not yield useful results.
+
+### Table
+
+Table charts display the results of a [Discover Query](/product/discover-queries/). This visualization is well suited to showing key fields and related aggregates. For example “duration percentiles for the most frequently visited transaction names”. You can also choose to display [issues](/product/issues/) data in a table visualization using the [dataset selector](/product/dashboards/widget-builder/#choose-your-dataset).
+
+### World Map
+
+Events are queried using the given function and `geo.country_code`. The results are plotted as a density value on a world map. An example scenario would be “in which countries are users experiencing the most errors”. You can zoom the map using the mouse wheel, or pan the map by clicking and dragging the map around.
+
+### Big Number
+
+A big number visualization displays the current value of a single function. This visualization is well suited for high-level aggregates. An example scenario would be “p95 of all transactions”.
+
+## Choose your dataset
+
+In the dataset selection step, you can choose which type of data you would like to use in your widget. This data is classified into three different datasets:
```

Comment:
```suggestion
In the dataset selection step, choose which type of data you would like to use in your widget. This data is classified into two different datasets: [errors and transactions](#errors--transactions) and [issues](#issues).
```

Reply:
good catch! the UI will not have releases for now